Sources/Recycling/ListViews/AdManagement/Cells/UserAdManagementStatisticsCell.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Copyright © FINN.no AS. All rights reserved.
+//
+
+public class UserAdManagementStatisticsCell: UITableViewCell {
+    public var itemModels = [StatisticsItemModel]() {
+        didSet { updateStackViewContent() }
+    }
+    private var stackView: UIStackView = {
```

Comment:
Can we add `lazy` between `private` and `var` and space between the two properties.

Reply:
Yes, we _should._ 😊